Testscripts/Linux/Test_LIS_disk_check.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# Source utils.sh
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+#Give the file size in K,M,G
+PrepareForTest() {
+	lis_tarball_link=$1
+	#For this test, we're expecting VHD with no LIS installed.
+	#If LIS is installed, then uninstall it.
+	if [ $(which rpm | wc -l) -gt 0 ]; then
+		total_hyperv_packages=$(rpm -qa | grep "hyper-v" | wc -l)
+		if [ $total_hyperv_packages -gt 0 ]; then
+			hyperv_package_names=$(rpm -qa | grep "hyper-v" | xargs)
+			LogMsg "Uninstalling hyperv packages..."
+			yum -y remove $hyperv_package_names
+			LogMsg "Removed $total_hyperv_packages hyperv packages."
+		else
+			LogMsg "LIS not found. (As expected)"
+		fi
+                lis_tarball_name=${lis_tarball_link##*/}
+                LogMsg "Downloading $lis_tarball_link"
+                wget $lis_tarball_link
+		LogMsg "Extracting $lis_tarball_name ..."
+		tar xzf $lis_tarball_name
+		LogMsg "Ready for the test."
+	else
+		LogMsg "Unsupported distro. Skipping."
+		SetTestStateSkipped
+		exit 0
+	fi
+}
+
+CreateFile() {
```

Comment:
this seems generic enough to worth be added to utils.sh, but please check first if something similar might already be there